File: Python/1020.py
import math

n = int(input())

y = math.floor(n / 365)
y_r = n % 365

m = math.floor(y_r / 30)
d = y_r % 30

print('{} ano(s)'.format(y))
print('{} mes(es)'.format(m))
print('{} dia(s)'.format(d))

File: JavaScript/1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var n = parseInt(lines.shift());

var y = Math.floor(n / 365)
var y_r = n % 365

var m = Math.floor(y_r / 30)
var d = y_r % 30

console.log(y + ' ano(s)')
console.log(m + ' mes(es)')
console.log(d + ' dia(s)')